vault/changelog.early-seed.md

```diff
@@ -10,27 +10,34 @@ nav_exclude: true
+
+**BREAKING:** Handlebars are now set as the default template format. Users that were using <% %> syntax for template variables will no longer work. Please see the [[Migration|dendron://dendron.dendron-site/dendron.topic.templates.handlebars.migration]] guide to switch to handlebar based templates. 
+
+**DEPRECATED:** `Insert Note` command replaced by the `Apply Template` command to [[Create a template|dendron://dendron.dendron-site/dendron.topic.templates.handlebars#create-a-template]]. 
 
 ### Features
-- feat(structure): add set task status commands (#3033) @kaan ([[docs|dendron://dendron.dendron-site/dendron.ref.commands#set-task-status]])
-- feat(structure): add complete task commands (#3033) @kaan ([[docs|dendron://dendron.dendron-site/dendron.ref.commands#complete-task]])
-- feat(sync): obsidian import flow (#3014) @jonathan ([[docs|dendron://dendron.dendron-site/dendron.ref.commands#import-obsidian-vault]])
+- feat(views): recent workspaces panel (#3052) @jonathan ([[docs|dendron://dendron.dendron-site/dendron.topic.sidebar.recent-workspaces]])
 
 ### Enhancements
-- enhance(publish): make the private link colors customizable (#3035) @kaan ([[docs|dendron://dendron.dendron-site/dendron.topic.theme.publish.cook#change-the-color-of-a-private-link]])
-- enhance(views): configure creation of notes on click of graph node with `createStubs` #3032 @joshi ([[docs|dendron://dendron.dendron-site/dendron.topic.graph-view#create-stub]])
-- enhance(workspace): detect duplicate note id on lifecycle events (#2991) @hikchoi
+- enhance(cli): updated dendron lookup command (#3043) @kevin ([[docs|dendron://dendron.dendron-site/dendron.topic.notes.cli.lookup]])
+- enhance(views): add tooltip on hover for graph nodes (#3060) @joshi ([[docs|dendron://dendron.dendron-site/dendron.topic.graph-view#features]])
+- enhance(edit): this sets handlebars as the default template format (#3070) @kevin
+- enhance(edit): configure templateHierarchy when applying template (#3069) @kevin ([[docs|dendron://dendron.dendron-site/dendron.topic.templates.config.templateHierarchy]])
+- enhance(retrieve): faster link completions (#3044)  @kevin
+- enhance(edit): remove deprecated `Insert Note` command (#3083) @kevin
+- enhance(workspace): prompt to update seed vault configuration during sync if the seed configuration changed (#3080) @kaan
+- enhance(workspace): v100 landing page (#3103) @jonathan
 
 ### Fixes
-- fix(workspace): fix duplicated panel titles (#3016) @jonathan
-- fix(edit): email addresses and hash symbols inside words are parsed as tags (#3064) @kaan
-- fix(views): gracefully handle tree view sort error to avoid crashing the tree view (#3053) @hikchoi
\ No newline at end of file
+- fix(workspace): migrate to self contained vault causes `EBUSY` errors on Windows (#3067) @kaan
+- fix(refactor): updated refactor hierarchy message to include no. of files changed (#3078) @joshi
+- fix(views): double click issue on help and feedback panel (#3089) @jonathan
```

Comment:
can we update on what the double click issue was here?

Reply:
Does this work - fix(views): issue with help and feedback panel launching a link once even if was clicked multiple times?

Reply:
https://github.com/dendronhq/dendron/pull/3089